auth.py

```diff
@@ -16,12 +16,15 @@ def authenticate(url):
     s = requests.Session()
 
     # 1. Initiate sequence by querying the protected asset
-    s.get(url)
+    if("https://autolab.cs.cmu.edu/" in url):
+    	s.get(url, verify=False)
```

Comment:
Lolwut. Poor Autolab. Is there any way that you can work around this outside of the authentication function, i.e. authenticate to an http:// URL instead? I'm not a fan of having URL-specific hacks inside the auth function wherever possible. 

Alternatively, you could add an additional optional parameter dictionary to the authenticate function and then pass that in to the session call.